addon/usables/use-machine.js

```diff
@@ -0,0 +1,102 @@
+import { interpret } from 'xstate';
+import { setUsableManager } from 'ember-usable';
+import { later, cancel } from '@ember/runloop';
+import { tracked } from '@glimmer/tracking';
+import { getOwner, setOwner } from '@ember/application';
+
+export class InterpreterService {
+  @tracked service;
+  @tracked currentState;
+
+  constructor(machine) {
+    this.machine = machine;
+  }
+
+  get state() {
+    return {
+      currentState: this.currentState,
+      send: this.service.send,
```

Comment:
what about transition?

Reply:
`this.service` refers to an interpreted machine - i.e. a xstate-[Interpreter](https://xstate.js.org/api/classes/interpreter.html). In contrast to a `Machine` an `Interpreter` has no `transition`-method only a `send`-method. You just send events to an interpreter and the interpreter then figures out if it needs to transition into a new state.

Reply:
I think I had just gotten confused with remembering my own code. I have a transition component action than just calls send